.github/workflows/tests.yml

```diff
@@ -63,3 +63,19 @@ jobs:
         run: |
           bin/rails assets:precompile
           bin/rails db:create db:schema:load db:seed spec
+  deploy:
+    runs-on: ubuntu-latest
+    needs: test
```

Comment:
if I'm reading this right, it means if the test step fails it won't deploy?  We should do the same for the lint step.

Reply:
that's right! the lint step is in a different workflow so I don't think we can refer to that easily. if we move linting here (we could) then yes we can do that

Reply:
something for the future, then